packages/next/src/client/components/react-dev-overlay/internal/container/Errors.tsx

```diff
@@ -273,9 +278,34 @@ export function Errors({
               </small>
               <VersionStalenessInfo versionInfo={versionInfo} />
             </LeftRightDialogHeader>
-            <h1 id="nextjs__container_errors_label">
-              {isServerError ? 'Server Error' : 'Unhandled Runtime Error'}
-            </h1>
+
+            <div className="nextjs__container_errors__error_title">
+              <h1
+                id="nextjs__container_errors_label"
+                className="nextjs__container_errors_label"
+              >
+                {isServerError ? 'Server Error' : 'Unhandled Runtime Error'}
+              </h1>
+              <span>
+                {error.stack && (
+                  <CopyButton
+                    data-nextjs-data-runtime-error-copy-stack
+                    actionLabel="Copy error stack"
+                    successLabel="Copied"
+                    content={error.stack}
+                  />
+                )}
+                {debugInfo?.devtoolsFrontendUrl && (
```

Comment:
What happens if you copy it in Firefox? Just good to know. Ideally we'd disable it in browser that don't support it but I think there's no accurate way to do that.

Reply:
Good catch, it won't work if you paste it into firefox browser. I'll add some filter